lib/data_models/genre.dart

```diff
@@ -0,0 +1,16 @@
+class Genre {
+  final int id;
+  final String name;
+
+  Genre ({
+    required this.id,
+    required this.name,
+  });
+
+  factory Genre.fromJson(Map<String, dynamic> genresJson) {
+    return Genre(
+      id: genresJson['id'] as int,
+      name: genresJson['name'] as String,
```

Comment:
You don't need to add the type cast operator

Reply:
Okay, fixed.